File: verify_sovereign.py
from playwright.sync_api import sync_playwright
import time

def verify():
    with sync_playwright() as p:
        browser = p.chromium.launch(headless=True)
        page = browser.new_page()
        page.set_viewport_size({"width": 1280, "height": 1000})
        page.goto("http://localhost:8002")
        # Wait for assets to load
        time.sleep(5)
        # Move mouse to trigger magnifier effect
        page.mouse.move(640, 500)
        time.sleep(1)
        page.screenshot(path="final_sovereign_v2.png")
        browser.close()

if __name__ == "__main__":
    verify()

File: verify_sovereign.js
import { chromium } from 'playwright';

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.setViewportSize({ width: 1280, height: 1000 });
  await page.goto('http://localhost:8002');
  // Wait for images and animations
  await page.waitForTimeout(5000);
  await page.mouse.move(640, 500); // Trigger magnifier
  await page.waitForTimeout(1000);
  await page.screenshot({ path: 'final_sovereign_magnifier.png', fullPage: false });
  await browser.close();
})();
